packages/slice-machine/src/components/ContentTabs/ContentTabs.module.css

```diff
@@ -0,0 +1,66 @@
+.flex {
+  all: unset;
+  display: flex;
+}
+
+.column {
+  composes: flex;
+  flex-direction: column;
+}
+
+.root {
+  composes: column;
+  background-color: var(--grey1);
+  min-height: 0;
+}
+
+.list {
+  composes: flex;
+  border-bottom: 1px solid var(--grey5);
+  padding-inline: 16px;
+}
+
+.trigger {
+  composes: column;
+  color: var(--grey11);
+  cursor: pointer;
+  padding-block: 8px;
+  position: relative;
+
+  &:not(:last-child) {
+    margin-right: 16px;
+  }
+
+  &[data-state="active"] {
+    color: var(--purple9);
+
+    &::after {
```

Comment:
💡 #suggestion

I checked the `editor-ui` and for very specific box-shadow they just are ignoring manually the rule. This ensures consistency by default, and you have to explicitly ignore to override. I think it's what we want here, WDYT?

Example: https://github.com/prismicio/editor/blob/main/packages/editor-ui/src/components/ColorInput/ColorSlider.module.css

Reply:
I guess no need of the position: relative if we change that, right?

Reply:
Oh cool, I missed that in the `editor-ui` repo. I had to use a `stylelint-disable` rather than `stylelint-disable-next-line` because Prettier formats the property into two lines. Should be good now!